Mage.Sets/src/mage/sets/MurdersAtKarlovManor.java

```diff
@@ -12,7 +12,7 @@
  */
 public final class MurdersAtKarlovManor extends ExpansionSet {
 
-    private static final List<String> unfinished = Arrays.asList("Aurelia's Vindicator", "Bubble Smuggler", "Coveted Falcon", "Crowd-Control Warden", "Expose the Culprit", "Fugitive Codebreaker", "Lumbering Laundry", "Museum Nightwatch", "Pyrotechnic Performer", "Unyielding Gatekeeper");
+    private static final List<String> unfinished = Arrays.asList("Aurelia's Vindicator", "Bubble Smuggler", "Coveted Falcon", "Crowd-Control Warden", "Expose the Culprit", "Fugitive Codebreaker", "Lumbering Laundry", "Museum Nightwatch", "Unyielding Gatekeeper");
```

Comment:
hm, not specifically related to this PR, but I'm pretty sure this unfinished can be removed now that Disguise is implemented. The reason for it is when cards are being partially implemented (missing an unimplemented keyword mechanic but with all other code present).

Reply:
There are must be only 3 cards with cloak [from MKC](https://scryfall.com/search?q=oracle%3Acloak+set%3Amkc&unique=cards&as=grid&order=name) and 5 cards [from MKM](https://scryfall.com/search?q=oracle%3Acloak+set%3Amkm&unique=cards&as=grid&order=name)